.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Ya I'm with @Glavin001 on this and I do think that we should be put IDE dot files in our `.gitignore`. Also gets hairy as more contributors come to the mix.